lib/Limitation/Mapper/OwnerLimitationMapper.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\Limitation\Mapper;
+
+use eZ\Publish\API\Repository\Values\User\Limitation;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class OwnerLimitationMapper extends MultipleSelectionBasedMapper
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    public function __construct(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    protected function getSelectionChoices()
+    {
+        return [
+            1 => $this->translator->trans('policy.limitation.owner.self', [], 'ezrepoforms_role'),
+            // 2: "Session" is not supported yet, see OwnerLimitationType
+            //2 => $this->translator->trans('policy.limitation.owner.session', [], 'ezrepoforms_role'),
```

Comment:
Please don't commit commented out code. It's fine to have a comment (should be above `return` statement), with what it should be, but commented out code is evil :wink: